source/operations/install-deploy-manage/deploy-minio-tenant-helm.rst

```diff
@@ -76,64 +89,113 @@ You can modify the Operator deployment after installation.
 
-   Use ``kubectl port-forward`` to temporarily forward traffic from the MinIO pod to your local machine:
+#. Configure Network Encryption
+
+   The MinIO Tenant CRD provides the following fields from which you can configure tenant TLS network encryption:
+
+   .. list-table::
+      :header-rows: 1
+      :widths: 30 70
+
+      * - Field
+        - Description
+
+      * - ``tenant.certificate.requestAutoCert``
+        - Enables or disables MinIO :ref:`automatic TLS certificate generation <minio-tls>`
+
+      * - ``tenant.certificate.certConfig``
+        - Controls the settings for :ref:`automatic TLS <minio-tls>`.
+          Requires ``spec.requestAutoCert: true``
+
+      * - ``tenant.certificate.externalCertSecret``
+        - Specify one or more Kubernetes secrets of type ``kubernetes.io/tls`` or ``cert-manager``.
+          MinIO uses these certificates for performing TLS handshakes based on hostname (Server Name Indication).
+
+      * - ``tenant.certificate.externalCACertSecret``
+        - Specify one or more Kubernetes secrets of type ``kubernetes.io/tls`` with the Certificate Authority (CA) chains which the Tenant must trust for allowing client TLS connections.
+
+#. Configure MinIO Environment Variables
+
+   You can set MinIO Server environment variables using the ``tenant.configuration`` field.
+
+   The field must specify a Kubernetes opaque secret whose data payload ``config.env`` contains each MinIO environment variable you want to set.
+
+   The YAML includes an object ``kind: Secret`` with ``metadata.name: storage-configuration`` that sets the root username, password, erasure parity settings, and enables Tenant Console.
+
+   Modify this as-needed to reflect your Tenant requirements.
+
+#. Deploy the Tenant
+
+   Use ``helm`` to install the Tenant Chart using your ``values.yaml`` as an override:
 
    .. code-block:: shell
       :class: copyable
 
-      kubectl --namespace MINIO_TENANT_NAMESPACE port-forward svc/MINIO_TENANT_NAME-console 9443:9443
-   
-   .. note::
-      
-      To configure long term access to the pod, configure :kube-docs:`Ingress <concepts/services-networking/ingress/>` or similar network control components within Kubernetes to route traffic to and from the pod.
-      Configuring Ingress is out of the scope for this documentation.
+      helm install \
+        --namespace minio-tenant-1 \
+        --create-namespace \
+        --values values.yaml \
+        tenant minio/tenant
+
+   You can monitor the progress using the following command:
 
-#. Login to the MinIO Console
+   .. code-block:: shell
+      :class: copyable
 
-   Access the Tenant's :ref:`minio-console` by navigating to ``http://localhost:9443`` in a browser.
-   Log in to the Console with the default credentials ``myminio | minio123``.
-   If you modified these credentials in the ``values.yaml`` specify those values instead.
+      watch kubectl get all -n minio-tenant
```

Comment:
```suggestion
      watch kubectl get all -n minio-tenant-1
```

Reply:
Not sure if you want to use `MINIO_TENANT_NAMESPACE` and `MINIO_TENANT_NAME` nomenclature, but we should be consistent.